images/manageiq-base/Dockerfile

```diff
@@ -105,9 +105,17 @@ RUN source /etc/default/evm && \
     if [ ${ARCH} = "ppc64le" ] ; then gem install sassc  -- --disable-march-tune-native ; fi && \
     bundle install && \
     rake tmp:clear evm:compile_sti_loader log:clear && \
-    find ${RUBY_GEMS_ROOT}/gems/ -name .git | xargs rm -rvf && \
+    RUBY_GEMS_ROOT=$(gem env gemdir) && \
```

Comment:
Should this be in the `ENV` command above or is it just for the cleanup?

Reply:
It's for clean up only, so I didn't want it to persist.